.idea/misc.xml

```diff
@@ -4,7 +4,7 @@
     <option name="noGCSettingWarning" value="true" />
   </component>
   <component name="ExternalStorageConfigurationManager" enabled="true" />
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_17" default="true" project-jdk-name="Android Studio default JDK" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_11" default="true" project-jdk-name="Android Studio default JDK" project-jdk-type="JavaSDK">
```

Comment:
Why did you switch this?

Reply:
Thank you for pointing this out, this must have been done by my android studio sneakily. I will revert this change.

Reply:
I have now reverted this